apps/gallery/style/info.css

```diff
@@ -31,7 +31,7 @@
   line-height: 1;
   font-weight: 400;
   margin: 0;
-  padding: 0 0 1.2rem 1rem;
+  padding: 0 1rem 1.2rem 1rem;
```

Comment:
we are changing padding-right here from 0 to 1rem to give 1 rem space from right edge in RTL mode